test_scripts/Defects/7_1/1597_3_AlertManeuver_Unsupported_Speech_Type.lua

```diff
@@ -0,0 +1,85 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/SmartDeviceLink/sdl_core/issues/1597
+--
+-- Description:
+-- HMI responds with UNSUPPORTED_RESOURCE to Speak component of AlertManeuver
+--
+-- Preconditions:
+-- 1) Clear environment
+-- 2) SDL, HMI, Mobile session started
+-- 3) Registered app
+-- 4) Activated app
+--
+-- Steps: 
+-- 1) Send AlertManeuver mobile RPC from app with ttsChunks, HMI responds with UNSUPPORTED_RESOURCE to 
+--    Speak request
+--
+-- Expected:
+-- 1) App receives AlertManeuver response with info from Speak response, WARNINGS result code, 
+--    and success=true
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require("user_modules/sequences/actions")
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Functions ]]
+local function pTUpdateFunc(tbl)
+  local newGroup = {
+    rpcs = {
+      AlertManeuver = {
+        hmi_levels = {"BACKGROUND", "FULL", "LIMITED", "NONE"}
+      }
+    }
+  }
+  tbl.policy_table.functional_groupings["NewTestCaseGroup"] = newGroup
+  tbl.policy_table.app_policies[config.application1.registerAppInterfaceParams.fullAppID].groups = {"Base-4", "NewTestCaseGroup"}
+end
+
+local function sendOnSystemContext(ctx, pWindowId, pAppId)
+  if not pWindowId then pWindowId = 0 end
+  if not pAppId then pAppId = 1 end
+  common.getHMIConnection():SendNotification("UI.OnSystemContext",
+  {
+    appID = common.getHMIAppId(pAppId),
+    systemContext = ctx,
+    windowID = pWindowId
+  })
+end
+
+local function sendAlertManeuver()
+  local cid = common.getMobileSession():SendRPC("AlertManeuver", {
+    ttsChunks = { 
+      { type = "LHPLUS_PHONEMES", text = "phoneme" }
+    }
+  })
+  common.getHMIConnection():ExpectRequest("Navigation.AlertManeuver")
+  :Do(function(_, data)
+      common.run.runAfter(function()
+        common.getHMIConnection():SendResponse(data.id, data.method, "SUCCESS", {})
+      end, 2000)
+    end)
+  common.getHMIConnection():ExpectRequest("TTS.Speak")
+  :Do(function(_, data)
+      common.run.runAfter(function()
+        common.getHMIConnection():SendError(data.id, data.method, "UNSUPPORTED_RESOURCE", "Error message from HMI")
+      end, 1000)
+    end)
+  common.getMobileSession():ExpectResponse(cid, { success = true, resultCode = "WARNINGS", info = "Error message from HMI" })
```

Comment:
The test seems to fail on my end with this error
```
--- Test -------
[13:15:20, 909] Alert_Maneuver
Response to 4: ValidIf:
types of data payload.info:expected string, actual type:nil
```

I think it's related to this line. If I remember correctly you cannot send an info param in a successful response to mobile

Reply:
Seems I never pushed one of my fixes to the original Core PR, it should be passing with the latest version of the Core branch now